src/migration.ts

```diff
@@ -102,10 +104,13 @@ export class Migration {
     }
     let db: string | Db;
     if (typeof (this.options.db) === 'string') {
-      const client = await MongoClient.connect(this.options.db, {
-        useNewUrlParser: true,
-      });
-      db = client.db();
+      const client = await MongoClient.connect(
+        this.options.db,
+        this.options.dbOptions || {
```

Comment:
We should keep `useNewUrlParser: true` as the default so I'd suggest merging `{ useNewUrlParser: true }` into the `this.options.dbOptions` if `this.options.dbOptions.useNewUrlParser` is null/undefined.

Reply:
Adjusted, see PR update.